.env.development

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://localhost:5432/moncomptepro
```

Comment:
@rdubigny This is temporally require to make "npm test" happy as it's importing the global config/env.ts